content/docs/use-cases/ci-cd-ml.md

```diff
@@ -0,0 +1,70 @@
+# Continuous Integration and Deployment for Machine Learning
+
+MLOps and DataOps apply DevOps methodologies to machine learning (ML) and data
+management. The fields are rapidly evolving beyond a mere set of best practices,
+and now encompass the entire product lifecycle management. A core component is
+the automation of testing and deployment: CI/CD (continuous integration and
+delivery).
+
+CI/CD is easy to set up with practically any Git repository. Some example
+scenarios are:
+
+- Agile development
+  - Automating quality assurance (QA) means teams can confidently deploy new
+    versions several times a day and even before the weekend without fear of
+    bugs/regressions
+- Easy testing
+  - When new data is pushed in a pull request (PR), validation tests
+    automatically run in the cloud (without each collaborator setting up
+    potentially inconsistent local testing environments)
+  - When a new model is pushed in a PR, reports (on performance against the
+    current deployed models) are automatically generated and posted as a PR
+    comment with graphs and tables
+- Continuous machine learning (CML): online training
```

Comment:
💅 What does the ": online training" part mean? Can it go under the bullet for style consistency/readability?

Reply:
online is a pun here really, meaning:

1. in the cloud, somewhere ~over the rainbow~ in the ephemeral interwebz
2. [refining rather than retraining as new data becomes available](https://en.wikipedia.org/wiki/Online_machine_learning)

Reply:
_online training_ is too vague and a search yields coursera and udemy, rather than ML and AI. 

_building models on the cloud_ is probably better.

Reply:
reworded